integration/copy_test.go

```diff
@@ -662,3 +662,66 @@ func verifyManifestMIMEType(c *check.C, dir string, expectedMIMEType string) {
 	mimeType := manifest.GuessMIMEType(manifestBlob)
 	c.Assert(mimeType, check.Equals, expectedMIMEType)
 }
+
+const (
+	regConf    = "/etc/containers/registries.conf"
+	regConfBak = "/etc/containers/registries.conf.bak"
+)
+
+func (s *SkopeoSuite) injectRegistriesConf(c *check.C) string {
```

Comment:
I’d prefer adding a `--registries.conf` option that sets `SystemRegistriesConfPath`, and calling `skopeo --registries.conf $fixture`. Surprisingly little of the existing code requires changing system-wide files the way this code does.

Reply:
Buildah has --registries-conf, although I prefer to keep these  hidden in those tools.

Reply:
Sure, the dash works just as well and consistency is good.

Reply:
I'm not sure if adding a new command line argument is nice just for testing purposes. @vrothberg WDYT?

Reply:
I think that it can be a nice addition (also for debugging). But as Dan mentioned, it should be hidden to avoid too much creativity.

Note that there is some selfish motivation behind: having such a flag saves me time as I don't need to copy the registries.conf around :^)